restucture debug handle

Gasoonjia opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #7197

Differential Revision: [D66622890](https://our.internmc.facebook.com/intern/diff/D66622890/)

Gasoonjia committed:
restucture debug handle



Differential Revision: [D66622890](https://our.internmc.facebook.com/intern/diff/D66622890/)

[ghstack-poisoned]

jerryzh168 commented on exir/graph_module.py:
```diff
@@ -68,3 +68,25 @@ def get_control_flow_submodules(
+
+# TODO(gasoonjia): remove this and leverage core pytorch bfs_trace_with_node_process after code freeze
+def bfs_trace_with_node_process(
+    gm: torch.fx.GraphModule, node_op: Callable[[torch.fx.Node], None]
```
this one takes `gm`? I remember another one takes `ep`? maybe align?

Gasoonjia replied:
There're two `bfs_trace_with_node_process`, one is under ao, the other is under et.
ao's take ep and gm while et's only takes gm.

Gasoonjia committed:
Update on "restucture debug handle"


Differential Revision: [D66622890](https://our.internmc.facebook.com/intern/diff/D66622890/)

[ghstack-poisoned]

Gasoonjia commented:
@pytorchbot label "topic: not user facing"

Gasoonjia committed:
Update on "restucture debug handle"


Differential Revision: [D66622890](https://our.internmc.facebook.com/intern/diff/D66622890/)

[ghstack-poisoned]
